### PR TITLE
Bump version to 1.5 (compiler)/0.5 (services)

### PR DESCRIPTION
**Pull request by DanielRosenwasser**

### CODE REVIEW

**Comment by DanielRosenwasser** [comment]
@mhegazy

**Comment by sheetalkamat** [comment]
:+1:

**Comment by clausreinke** [comment]
version bump + fresh LKG should always go together, right?

File: package.json [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
+        "mocha": "latest",
+        "chai": "latest",
+        "browserify": "latest",
+        "istanbul": "latest",
+        "codeclimate-test-reporter": "latest"
+    },
+    "scripts": {
+        "test": "jake generate-code-coverage"
+    }
 }
```
Comment (basarat) [inline package.json]: good call. Most json writing tools (e.g. npm) will write 4 spaced `.json` files.

Comment (DanielRosenwasser) [inline package.json]: Good, I was just going on the simple fact that tabs are one of the 3 worst things on Earth.

**Comment by DanielRosenwasser** [comment]
@clausreinke we usually don't have an LKG in a PR.

**Comment by clausreinke** [comment]
@DanielRosenwasser okay, thanks. And the git tag?-)

**Comment by DanielRosenwasser** [comment]
We're not officially at 1.5 so we're not tagging yet. Basically, the way we've done it is we'll decide a good point for 1.5 to stabilize on and then snap off into a new branch (`release-1.5` presumably). Fixes and committed features go into that branch and are merged back into `master`. When we decide to officially release, that  branch will get a tag. Do you have an alternate scheme in mind?

**Comment by clausreinke** [comment]
Ah, I just saw the unadorned version and expected a tag/release. 

I've tried your approach in the past and found it confusing: once one installs the package, the git context is gone, and one is left wondering which version it was that failed tests or created bug reports. 

Since then, I have added two habits: 
1. add a label to the version indicating the branch (I usually have a branch for testing, and not as many features under concurrent development as you do) 
2. add a build number to the version (sometimes, I've also used the output of `git describe --tags`, but a simple number suffices and is easier to maintain)

I've found these more descriptive versions quite helpful in avoiding confusion (I also use them as output for `--version` queries) - the only caveat: it can be easy to create a version label that not all tools are happy with.

**Pull request merged**